rustfmt.toml

```diff
@@ -1,20 +1,20 @@
-format_code_in_doc_comments = true
-match_block_trailing_comma = true
+comment_width = 100
```

Comment:
This one was actually formatted by length, so it was a nice ladder.

Reply:
Lol.
I can disable the alphabetical sorting?
But I think that's not a small part of the whole point here.

I prefer alphabetical ordering for ease-of-use over "looking nice" anyway.